CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+## How to contribute to ezplatform.com
```

Comment:
@SylvainGuittard what do you think about the content of this file ?

If you want something else just right the text and I'll change it.